src/App.js

```diff
@@ -1,26 +1,62 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/Header';
+import Main from './components/Main';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// import Footer from "./components/Footer";
+
+
+class App extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            itemsList: []
+        }
+    }
+
+    changeItem = (item) => {
+        console.log("Checked", item)
+    };
+
+    addItem = (item) => { // Просьба подсказать, как в таком случае создавать элементы у которых можно задать свой id,
+        // и остальные флаги с помощью которых смогу далее реализовать все остальные методы. В этом методе не получается(
+        // Так как изначально не продумала всю архитектуру приложения, теперь на каждом шагу все ломаю((
+        if (!item) {
+            return
+        }
+        this.setState((prev) => {
+                return {
+                    itemsList: [
+                        ...prev.itemsList,
+                        item
+                    ]
+                }
+            }
+        )
+    };
+
+    removeItem = (item) => {
+        this.setState((prev) => {
+                let copyList = [...prev.itemsList];
+                copyList.splice(item, 1);
+                return {itemsList: copyList};
```

Comment:
```suggestion
                return {
                  itemsList: prev.itemsList.filter((item) => item.id !== id);
                };
```
this way you can get rid of lines 40 and 41